unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/SceneInfo/SceneLimitInfoController.cs

```diff
@@ -90,6 +90,8 @@ void SetFillInfo()
             colorToUse = highFillColor; 
         foreach(Image img in fillsImgs)
```

Comment:
Not entirely clear on what `fillsImgs` is supposed to mean. Can we rename this to something more obvious?